src/AddItem/AddItem.tsx

```diff
@@ -1,12 +1,50 @@
 import { useState, memo } from 'react';
 import type { FC } from 'react';
 import type { Element } from 'types'
-import s from './index.module.css'
+import styled from 'styled-components'
 
 type AddItemProps = {
     setTodoList: any
     todoList: Array<Element>
 }
+const NewElement = styled.div`
+    white-space: nowrap;
+    padding: 10px 20px;
+    border: none;
+    border-bottom: solid 3px #aaa;
+    background-color: #fff;
+    margin: 20px 0;
+    border-radius: 5px;
+    box-shadow: 0px 00px 50px #0005;
+`;
+const Input = styled.input`
+background-color: #0000;
+border: none;
+font-family: 'Courier New', Courier, monospace;
+font-size: 0.9rem;
+padding: 0 20px;
+&:focus {
+    outline: none;
+}
+&:placeholder {
```

Comment:
`&::placeholder` -- two '::' because placeholder is a pseudo-element

Reply:
you don't need to specify the same styles for placeholder. placeholder inherits font styles from the input. as I know